Sources/web3swift/Web3/Web3+HttpProvider.swift

```diff
@@ -52,4 +56,36 @@ public class Web3HttpProvider: Web3Provider {
         }
         attachedKeystoreManager = manager
     }
+
+    static func post<T: Decodable, U: Encodable>(_ request: U, providerURL: URL, session: URLSession) async throws -> T {
+
+        let requestData = try JSONEncoder().encode(request)
+        var urlRequest = URLRequest(url: providerURL, cachePolicy: .reloadIgnoringCacheData)
+        urlRequest.httpMethod = "POST"
+        urlRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        urlRequest.setValue("application/json", forHTTPHeaderField: "Accept")
+        urlRequest.httpBody = requestData
+
+        let (data, _) = try await session.data(for: urlRequest)
```

Comment:
This method available only on macOS 12, which is brings lack of backward capability. Please replace this method to something that available in macOS 10.15 and iOS 13 versions.

Reply:
updated to resolve this